doc/ARCHITECTURE.md

```diff
@@ -1,29 +1,83 @@
+## Architecture
+The BO1LER, a fully customisable design system (DS), is developed using the [Lit](https://lit.dev/docs/) template and Figma. The design system serves as a core unit for UI components, styles, and guidelines, ensuring consistency and efficiency across our applications.
 
 ## Content
-
 - [Architecture](#architecture)
   - [Content](#content)
   - [Overall Structure](#overall-structure)
+     - [File Structure](#file-structure)
+     - [Version Control](#version-control)
+     - [Documentation and Testing](#documentation-and-testing)
+     - [CI/CD](#ci-cd)
   - [Technical Decisions](#technical-decisions)
-  - [APIs](#apis)
+     - [Web Components](#web-components)
+     - [Styles](#styles)
+     - [Guidelines](#guidelines)
+     - [Figma](#figma)
+  - [Contributing](#contributing)
 
 ## Overall Structure
+The entire code is developed in TypeScript, which facilitates type checking and offers additional functionalities for detecting errors early in the development process. Besides configuration files, the 'Packages' directory holds essential content. Within this directory are subfolders named according to their contents.
+
+### File Structure
+Our project has the following file structure:
+
+    └── .github             // GitLab CI/CD pipeline
+    └── .husky              // Git hooks configuration files and scripts
+    └── .vscode             // VS code extensions and settings json
+    └── .yarn               // Yarn releases folder
+    └── doc                 // The documentation skeleton
+    └── packages
+        └── assets               // Collection of resources, such as images and other files utilized in B01LER
+        └── eslint-config-boiler // Specific ESLint configuration for this project
+        └── figma-design-tokens  // Design tokens configuration files and exported token json files
+        └── icons                // Icon configuration files and a collection of icon files in svg format
+        └── js-example-app       // Source code for js testing app for B01LER DS
+        └── storybook            // Configuration files and compiling tools for Storybook
+        └── ui-library
+            └── .storybook          // Configuration files for Storybook environment, assets, fonts and theme
+            └── src
+                └── components         // Collection of individual B01LER DS components
+                └── foundation         // Collection of generated token files and component and semantic tokens
+                └── globals            // Exports of constants and types
+                └── util               // Collection of utility or helper functions
+    └── CODE_OF_CONDUCT.md  // Our Code of conduct
+    └── CONTRIBUTING.md     // The file you have open right now
+    └── LICENCE.md          // Software license that governs the use, distribution, and modification of B01LER
+    └── README.md           // Introductory documentation for B01LER
+
```

Comment:
Do these files need to be nested within Docs? And should it include the Architecture, Deployment, GetStarted, Glossary, ProjectBackground, Security, Support and Testing md files here too?

Reply:
You made me realize, that we need to update this a bit, so I started a new review below, commenting on this section as well